client/src/pages/Prospecting/Prospecting.js

```diff
@@ -1,685 +1,536 @@
-                                    ) {
-                                        return;
-                                    }
-                                    handleRefresh();
-                                }}
-                                color="primary"
-                                size="small"
-                            >
-                                <RefreshIcon />
-                            </IconButton>
-                        </Tooltip>
-                    </Box>
-                </Box>
-
-                {error ? (
-                    <Alert severity="error">{error}</Alert>
-                ) : isSummary && !summaryLoading ? (
-                    <ProspectingSummary
-                        period={period}
-                        summaryData={summaryData}
-                    />
-                ) : (
-                    <>
-                        <CustomTable
-                            tableData={{
-                                columns: columnShows,
-                                data: detailedActivationData.map((item) => ({
-                                    ...item,
-                                    "account.name": (
-                                        <Link
-                                            href={`${instanceUrl}/${item.account?.id}`}
-                                            target="_blank"
-                                            rel="noopener noreferrer"
-                                        >
-                                            {item.account?.name || "N/A"}
-                                        </Link>
-                                    ),
-                                    "opportunity.name": item.opportunity ? (
-                                        <Link
-                                            href={`${instanceUrl}/${item.opportunity.id}`}
-                                            target="_blank"
-                                            rel="noopener noreferrer"
-                                        >
-                                            {item.opportunity.name || "N/A"}
-                                        </Link>
-                                    ) : (
-                                        "N/A"
-                                    ),
-                                })),
-                                selectedIds: new Set(),
-                                availableColumns: tableColumns,
-                            }}
-                            paginationConfig={{
-                                type: "server-side",
-                                totalItems: totalItems,
-                                page: page,
-                                rowsPerPage: rowsPerPage,
-                                onPageChange: handlePageChange,
-                                onRowsPerPageChange: handleRowsPerPageChange,
-                            }}
-                            sortConfig={{
```

Comment:
@ter1203 can you include this in the CustomTable within the AccountDetail compoent? You should be able to prop drill it exactly like you're doing with pagination config, when done correctly it should result in detail table rows getting sorted when you click a column.

Reply:
Ok, got it.